### DISCUSSION TITLE
RDS RemoteApp issue after self-hosted Teleport deployment on Windows Server

### DISCUSSION THREAD

**Discussion by sgaudl** [gravitational/teleport#47941] [Q&A]
Hi,
I've just deployed Teleport (by GPO) on Windows Server joined to Active Directory and connection through Teleport is working well.
In the same time, users had opened cas at support 'cause the were not able to connect to their Remotte App published by the RDS service.
After debugging, I've found events on the broker :
Security : 
- 4624 Logon => session opneing OK
- 4634 Logoff => in the same second, session closed.
Remote Destop Service : 
- 1306 : service broker client anable to redirect the user, error: bad parameter

If I unlink the GPO from the OU and make a gpupdate on the server, the RemoteApp works again.

The polology is like : 
broker server in an OU
app server in another OU

If I apply the GPO only on the app OU, the connection issue is present with the RemoteApp but I'm able to connect to the serveur on RDP and through Teleport.

I've found nothing over documentation about RDS RemoteApp self-hosted.
Any idea?

**Comment by webvictim** [comment]
We haven't tested or used RDS RemoteApp with Teleport. It sounds like some kind of authentication conflict. If you're a Teleport Enterprise customer, we recommend opening a support ticket to get assistance. Otherwise, please raise a [bug report](https://github.com/gravitational/teleport/issues/new?template=bug_report.md) in this repo with comprehensive details of how to reproduce the issue.
